### PR TITLE
[OCRVS 6250] Feature flags

### PR DESCRIPTION
**Pull request by tahmidrahman-dsi**
https://github.com/opencrvs/opencrvs-farajaland/pull/806

### CODE REVIEW

File: packages/client/src/components/interface/Navigation.tsx [hunk 1]
```diff
@@ -553,7 +553,7 @@ const NavigationView = (props: IFullProps) => {
                       }}
                     />
                   )}
-                {window.config.EXTERNAL_VALIDATION_WORKQUEUE && (
+                {window.config.FEATURES?.EXTERNAL_VALIDATION_WORKQUEUE && (
```
Comment (rikukissa) [inline packages/client/src/components/interface/Navigation.tsx:556]: Is the `FEATURES:` key required or optional?

File: packages/client/src/views/RegisterForm/review/ReviewSection.tsx [hunk 2]
```diff
@@ -1752,7 +1752,7 @@ class ReviewSectionComp extends React.Component<FullProps, State> {
 
     const getSignature = (eventType: Event) => {
       if ([Event.Birth, Event.Death].includes(eventType)) {
-        if (!offlineCountryConfiguration.config.INFORMANT_SIGNATURE) {
+        if (!offlineCountryConfiguration.config.FEATURES.INFORMANT_SIGNATURE) {
```
Comment (rikukissa) [inline packages/client/src/views/RegisterForm/review/ReviewSection.tsx:1755]: What's the difference between referring through offlineCountryConfiguration and `window.config`?

**Review by rikukissa — APPROVED** [review]

File: packages/client/src/components/interface/Navigation.tsx (hunk 1 above)
Comment (tahmidrahman-dsi) [reply packages/client/src/components/interface/Navigation.tsx:556]: it is required indeed. It was a temporary change, I am going to correct it

File: packages/client/src/views/RegisterForm/review/ReviewSection.tsx (hunk 2 above)
Comment (tahmidrahman-dsi) [reply packages/client/src/views/RegisterForm/review/ReviewSection.tsx:1755]: Good catch @rikukissa I think there is no difference. But in my opinion, it should use `window.config` as a single source of truth of application config

Comment (rikukissa) [reply packages/client/src/views/RegisterForm/review/ReviewSection.tsx]: I agree 👍 easier to search and replace in the future

**Pull request merged**